socs/agents/hwp_supervisor/agent.py

```diff
@@ -118,12 +118,12 @@ class HWPClients:
 class IBootState:
     instance_id: str
     outlets: List[int]
+    driver_power_agent_type: Literal['iboot, synaccess']
```

Comment:
Can we change this to just `agent_type`? The `driver-power-agent-type` arg is for specifying the power-agent type of the HWP driver board, so in the IBootState class we don't need those first two descriptors.